showtimes_gql/mutations/src/projects.rs

```diff
@@ -729,7 +727,9 @@ async fn fetch_metadata_via_anilist(
                 aired_at: Some(current_time),
             });
 
-            current_time += jiff::SignedDuration::new(ONE_WEEK, 0);
+            current_time = current_time
+                .checked_add(1.weeks())
```

Comment:
This will fail. As mentioned, if you want to do calendar arithmetic, you need to use `Zoned`.

It only makes sense to use `Zoned` for something like this if "add 1 week" is something you want to have time zones taken into account for.